students/kevin_t/lesson9/cli_main.py

```diff
@@ -0,0 +1,81 @@
+from mailroom_OO import Donor as D
+from mailroom_OO import DonorCollection as DC
```

Comment:
package names should be all lower case `mailroom_oo` (file names)

also, I would avoid aliases generally (there are uses cases for this) - `Donor` is more clear than `D`